app/translations/es.json

```diff
@@ -317,7 +317,7 @@
   "history": {
     "shieldfunds-all": "Blindar Todos los fondos",
     "shieldfunds-transparent": "Blindar fondos Transparentes",
-    "shieldfunds-sapling": "Blindar fondos Sapling",
+    "shieldfunds-sapling": "Actualizar fondos a la última versión",
```

Comment:
I don't think actualizar is the correct term because you mean to upgrade and not to update 

![image](https://github.com/zingolabs/zingo-mobile/assets/484008/f8242ff5-3bb3-47bd-96ac-c24de620e6eb)

Reply:
Thanks @pacu. My feeling here point to: `Asignar`, I think is the best fit.